### ISSUE TITLE
release and distribution

### ISSUE THREAD

**Issue by kodedninja** [new-computers/seeder#9]
An issue for ideas about the distribution / release of this.

Currently the simplest way to install it is `install.sh`. Maybe an `.img` of Raspbian that's set up would be a simpler and zero-config way for someone who just wants to dedicate the Pi for seeding.

**Comment by callil** [comment]
yes - another consideration down the line is being able to push updates to people's devices they can choose to install if they would like.

**Comment by kodedninja** [comment]
Probably [Resin.io](https://resin.io/) will be our saver for this
